pkg/terraform/terraform.go

```diff
@@ -8,15 +8,20 @@ import (
 	"github.com/pkg/errors"
 )
 
+// DefaultWorkingDir is the default working directory for Terraform
+const DefaultWorkingDir = "/cnab/app"
```

Comment:
I don't think this is the right directory, shouldn't it be /cnab/app/terraform? I'm going off of what I learned talking with Josh when we reviewed the original PR, not any real terraform knowledge.

Or does something about this PR change it and I missed that? Or does terraform assume that it's being run in the same directory as a bunch of *.tf files? And if we change the default directory then we need to be passing in another flag with the directory containing those files? Sorry so many questions because I don't know terraform... 😀

Reply:
I'm... also not a tf pro :) But I believe we can choose the directory.  `/cnab/app` does currently work fine, but if we'd rather establish a convention, `/cnab/app/terraform` sounds appropriate.

On a related note, you'll notice I'm in the habit of supplying the working dir on each terraform action, even though the binary is being executed at `/` in the container.  This actually isn't quite the recommended way :)  To do it the suggested way, we'd want to execute the binary from the working directory itself, so all of the generated *.tf/.terraform assets co-exist with the assets we provide.  If we go this route, we'd make sure we `cd` to the proper directory (`/cnab/app/terraform` perhaps) and then we no longer append this dir to each command invocation.

Reply:
Terraform does assume its running in a directory containing .tf files and runs all of them

Reply:
oh, yeah I guess my vote is for making the default /cnab/app/terraform and then cding into it. Just because if I were doing things by hand, that's what I'd do. Seems like it would be easier for people to debug and think about it that way.

Reply:
Ok, updated.